lib/aemo/version.rb

```diff
@@ -24,7 +24,7 @@
 # @author Joel Courtney <euphemize@gmail.com>
 module AEMO
   # aemo version
-  VERSION = '0.1.41'.freeze
+  VERSION = '0.1.42'
```

Comment:
Why the removal of freeze again @jeznag ?

Reply:
![image](https://user-images.githubusercontent.com/1034890/33046584-b548e438-cea5-11e7-8eea-cc639e86f0d8.png)

Reply:
👌🏼